packages/material-ui/src/ClickAwayListener/ClickAwayListener.test.js

```diff
@@ -153,19 +138,24 @@ describe('<ClickAwayListener />', () => {
         </ClickAwayListener>,
       );
 
-      fireEvent.click(getByText('Outside a portal'));
+      userEvent.mousePress(getByText('Outside a portal'));
       expect(handleClickAway.callCount).to.equal(0);
 
-      fireEvent.click(getByText('Stop all inside a portal'));
+      userEvent.mousePress(getByText('Stop all inside a portal'));
       expect(handleClickAway.callCount).to.equal(0);
 
-      fireEvent.click(getByText('Stop inside a portal'));
+      userEvent.mousePress(getByText('Stop inside a portal'));
       // undesired behavior in React 16
       expect(handleClickAway.callCount).to.equal(React.version.startsWith('16') ? 1 : 0);
     });
 
-    ['onClick', 'onClickCapture'].forEach((eventListenerName) => {
-      it(`should not be called when ${eventListenerName} mounted the listener`, () => {
+    [
+      ['onClick', 'onClick'],
+      ['onClick', 'onClickCapture'],
+      // earliest possible event we call `onClickAway` + earliest possible event that can mount the CAL
+      ['onMouseDown', 'onMouseDownCapture'],
+    ].forEach(([mouseEvent, eventListenerName]) => {
```

Comment:
You have to pass `mouseEvent` to the ClickAwayListener or it will always listen for click events.

Reply:
Yes, I forgot to pass `mouseEvent`